pkgs/misc/vscode-extensions/vscode-lldb/lldb/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchFromGitHub, llvmPackages_8, cmake, python3, swig, libedit, breakpointHook }:
+
+let
+  rev = "7f0cd38dae5cd77ac58adf97896ac0f41af759fc";
+  sha256 = "1z45wz0cfghirp2pwj6q8yqzpaafx5m7h206aj1wnqqb1k7qvwx4";
+
+  inherit (llvmPackages_8) llvm clang-unwrapped;
+
+in stdenv.mkDerivation {
+  name = "lldb-rust-enabled";
+  version = "git-${rev}";
```

Comment:
The version should be `unstable-2018-12-06` per the [nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming).

Reply:
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".